Geometry/CommonDetUnit/interface/GeomDetLess.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TrackerGeometryBuilder_GeomDetLess_H
+#define TrackerGeometryBuilder_GeomDetLess_H
+
+#include "DataFormats/TrajectorySeed/interface/PropagationDirection.h"
```

Comment:
The BuildFile.xml for Geometry/CommonDetUnit should also be changed to depend upon DataFormats/TrajectorySeed.